### ISSUE TITLE
Discussion - Improve email templates

### ISSUE THREAD

**Issue by eoglethorpe** [the-deep/deeper#949]
@Mariam-Asatryan @AdityaKhatri  can you guys pls discuss what emails get sent out and could we get a touch up of the formatting?

![image.png](https://zube.io/files/deep/031e7dbe8ec7b9c62d35d1f752e96d41-image.png)

**Comment by mariam-yu** [comment]
@eoglethorpe sure. goes into the pile. We had a fruitful discussion on how to organize and prioritize the design pile. So, once that's set up, this should definitely be in there somewhere @AdityaKhatri

**Comment by AdityaKhatri** [comment]
Hi @Mariam-Asatryan @eoglethorpe , we can base the templates on these:
https://codedmails.com/

The templates here are tested with multiple email clients.

**Comment by mariam-yu** [comment]
@AdityaKhatri Good stuff. Are you guys working on this in parallel or it's parked a bit? We could tackle this in a few phases. 1 - let's discuss logically what type of emails get send out and with which content. You probably already have this, but we could review,, 2 - (probably a bit later on) we can provide with UI designs on how to present that.

**Comment by AdityaKhatri** [comment]
@Mariam-Asatryan , we are not currently working or creating new templates.
The different types of emails we send are:

1. Reset password email
- Contains the link to reset password
2. Project join request emails
- Contains which user wants to join what project with reason provided along buttons to either accept or reject users directly from email
3. Comment assignee and reply emails
- When a user is assigned to a comment thread, and if there are replies to the assigned comments
4. User activation email 
- When a user first registers and a link to set password is sent to the user
